.gitignore

```diff
@@ -6,7 +6,6 @@ node_modules
 
 # builds
 build
-dist
```

Comment:
How much work will it be to add a `build/{branch-name}` system, like in neo-ui we have?

Reply:
We shouldn't need a build system actually, the package.json in this branch already has a `prepare` script that should generate dist files before installing the dependency, but somehow this is not running. Spent a lot of time debugging and exploring other alternatives but could not get it to work. So, pushed the dist files as a workaround since the feature is critical and we won't be updating the dependency frequently.

Reply:
Please add a TODO in readme of this project

Reply:
Done